tox.ini

```diff
@@ -2,12 +2,27 @@
 envlist = testenv
 
 [testenv]
-deps = responses
+deps = aresponses
 commands = python -m unittest discover {posargs:tests/unit}
 
 [testenv:unit]
 commands = python -m unittest discover tests/unit
 
 [testenv:integration]
-passenv = POWERWALL_IP POWERWALL_PASSWORD
+passenv = POWERWALL_IP,POWERWALL_PASSWORD
 commands = python -m unittest discover tests/integration
+
+[testenv:lint]
+description = run linters
+skip_install = true
+deps =
+    black==22.12
+commands = black {posargs:.}
+
+[testenv:type]
+description = run type checks
+deps =
+    mypy>=0.991
+    {[testenv]deps}
+commands =
+    mypy {posargs:tesla_powerwall tests} --ignore-missing-imports
```

Comment:
This is already handled by pre-commit. Please remove this from tox and use pre-commit instead.